test/js/specs/happy.js

```diff
@@ -2,125 +2,424 @@ const path = require('path')
+      verificationComplete.verificationCompleteMessage.isDisplayed()
+      const verificationCompleteThankYou = verificationComplete.verificationCompleteThankYou.getText()
+      expect(verificationCompleteThankYou).to.equal(documentUploadLocale["complete"]["submessage"])
+      verificationComplete.verificationCompleteThankYou.isDisplayed()
+    })
+
+    it('should return no face found error for selfie', async () => {
+      goToPassportUploadScreen(`?async=false&language=&useWebcam=false`)
+      const input = documentUpload.getUploadInput()
+      input.sendKeys(path.join(__dirname, documentUpload.assignDir('passport.jpg')))
+      waitForUploadToFinish
+      documentUploadConfirmation.confirmBtn.click()
+      const inputSelfie = documentUpload.getUploadInput()
+      inputSelfie.sendKeys(path.join(__dirname, documentUpload.assignDir('llama.jpg')))
+      waitForUploadToFinish
+      documentUploadConfirmation.confirmBtn.click()
+      const noFaceError = documentUploadConfirmation.errorTitleText.getText()
+      expect(noFaceError).to.equal(documentUploadConfirmationLocale["errors"]["no_face"]["message"])
+      documentUploadConfirmation.errorTitleText.isDisplayed()
+      documentUploadConfirmation.errorTitleIcon.isDisplayed()
+      const noFaceInstruction = documentUploadConfirmation.errorInstruction.getText()
+      expect(noFaceInstruction).to.equal(documentUploadConfirmationLocale["errors"]["no_face"]["instruction"])
+      documentUploadConfirmation.errorInstruction.isDisplayed()
+    })
+
+    it('should return multiple faces error', async () => {
+      goToPassportUploadScreen(`?async=false&language=&useWebcam=false`)
+      const input = documentUpload.getUploadInput()
+      input.sendKeys(path.join(__dirname, documentUpload.assignDir('passport.jpg')))
+      waitForUploadToFinish
+      documentUploadConfirmation.confirmBtn.click()
+      const inputSelfie = documentUpload.getUploadInput()
+      inputSelfie.sendKeys(path.join(__dirname, documentUpload.assignDir('two_faces.jpg')))
+      waitForUploadToFinish
+      documentUploadConfirmation.confirmBtn.click()
+      const multipleFacesError = documentUploadConfirmation.errorTitleText.getText()
+      expect(multipleFacesError).to.equal(documentUploadConfirmationLocale["errors"]["multiple_faces"]["message"])
+      documentUploadConfirmation.errorTitleText.isDisplayed()
+      documentUploadConfirmation.errorTitleIcon.isDisplayed()
+      const multipleFacesInstruction = documentUploadConfirmation.errorInstruction.getText()
+      expect(multipleFacesInstruction).to.equal(documentUploadConfirmationLocale["errors"]["multiple_faces"]["instruction"])
+      documentUploadConfirmation.errorInstruction.isDisplayed()
+    })
+
+    it('should return glare detected message on front and back of doc', async () => {
+      driver.get(localhostUrl + `?async=false&language=&useWebcam=false`)
+      welcome.primaryBtn.click()
```

Comment:
You wouldn't need to repeat all of these lines if they were wrapped inside it's own function.